src/locales/ja.json

```diff
@@ -1,555 +1,555 @@
-  "No shipping label error received from carrier.": "No shipping label error received from carrier",
-  "No capacity sets the fulfillment capacity to 0, preventing any new orders from being allocated to this facility. Use the \"Reject all orders\" option in the fulfillment pages to clear your facilities fulfillment queue. To add a fulfillment capacity to this facility, use the custom option.": "No capacity sets the fulfillment capacity to 0, preventing any new orders from being allocated to this facility. Use the \"Reject all orders\" option in the fulfillment pages to clear your facilities fulfillment queue. {space}To add a fulfillment capacity to this facility, use the custom option.",
-  "No channels found": "No channels found",
-  "No new file upload. Please try again": "No new file upload. Please try again",
-  "No fulfillment capacity": "No fulfillment capacity",
-  "No keyword matches the search criteria.": "No keyword matches the search criteria.",
-  "No orders found.": "No orders found.",
-  "No orders are currently able to be shipped due to missing tracking codes.": "No orders are currently able to be shipped due to missing tracking codes.",
-  "No payments found": "No payments found",
-  "No reason": "No reason",
-  "No results found for . Try searching Open or Completed tab instead. If you still can't find what you're looking for, try switching stores.": "No results found for { searchedQuery }. Try searching Open or Completed tab instead.{ lineBreak } If you still can't find what you're looking for, try switching stores.",
-  "No results found for . Try searching In Progress or Open tab instead. If you still can't find what you're looking for, try switching stores.": "No results found for { searchedQuery }. Try searching In Progress or Open tab instead.{ lineBreak } If you still can't find what you're looking for, try switching stores.",
-  "No results found for . Try searching In Progress or Completed tab instead. If you still can't find what you're looking for, try switching stores.": "No results found for { searchedQuery }. Try searching In Progress or Completed tab instead.{ lineBreak } If you still can't find what you're looking for, try switching stores.",
-  "No ship groups found": "No ship groups found",
-  "No time zone found": "No time zone found",
-  "Not in Stock": "Not in Stock",
-  "Not-Authorized": "Not-Authorized",
-  "Not-Received": "Not-Received",
-  "No Capacity": "No Capacity",
-  "No packed shipments to ship for these orders": "No packed shipments to ship for these orders",
-  "Oldest to newest": "Oldest to newest",
-  "order": "order",
-  "of": "of",
-  "Off": "Off",
+  "ID cannot be more than 20 characters.": "IDは20文字以内でなければなりません。",
+  "Imported from Shopify": "Shopifyからインポート",
+  "is identified as unfulfillable. other containing this product will be unassigned from this store and sent to be rebrokered.": "{productName}は出荷不可能と判定されました。他の{orders}件の{orderText}がこのストアから解除され、再仲介のために送られます。",
```

Comment:
From this line we have removed a variable `{space}` in the translation, revert that change.

Reply:
There is only one line with {space} variable, line no. 263. Removing the {space} variable.

Reply:
I have asked to add the `{space}` back which was present before the translations, we do not want to remove/translate anything that was within `{}` on the left hand side of the translation.

Reply:
Added { space } variable for all the required places.